### ISSUE TITLE
cocoa pod installation not working on xcode 5.1

### ISSUE THREAD

**Issue by faizankhan1909** [aws/aws-sdk-ios#150]
Hi, 
   I am using xcode 5.1 and have integrated AWSiOSSDKv2 in a blank single view controller POC demo by following steps mentioned on this link:

https://github.com/aws/aws-sdk-ios

my project is compiling fine on xcode 6.0.1 but not working on xcode 5.1

UICKeyChainStore.h & UICKeyChainStore.m have 17 errors in total. The issue is somewhat compatibility with ios 7.

kSecAccessControlUserPresence is used in UICKeyChainStore.h but is not provided in security framework for ios 7

can you please guide me how to use the latest amazon SDK in my xcode 5.1 project?

**Comment by yosuke-matsuda** [comment]
You can update your Podfile to something like the following to use the older version of UICKeyChainStore that is compatible with Xcode 5:

    source 'https://github.com/CocoaPods/Specs.git'

    pod 'AWSiOSSDKv2', '2.0.15'

Why are you using Xcode 6 and 5 at the same time? Do you still have some projects that have not been converted for Xcode 6?

**Comment by faizankhan1909** [comment]
Hi, 
   Yes I have some projects that are not yet converted for xcode 6. I have made changes in my podFile and got the following error.

[!] Unable to satisfy the following requirements:

- `AWSiOSSDKv2 (= 2.0.15)` required by `Podfile`
- `AWSiOSSDKv2 (~> 2.0.16)` required by `AWSCognitoSync (1.0.8)`
- `AWSCognitoSync` required by `Podfile`
- `AWSCognitoSync (= 1.0.8)` required by `Podfile.lock`

so I have updated my pod file as follows:

source 'https://github.com/CocoaPods/Specs.git'

pod 'AWSiOSSDKv2' , '2.0.15'
pod 'AWSCognitoSync', '1.0.7'

The project is now compatible with xcode 5.1 and running fine on xcode 6.0.1 as well. Thank you for helping me out here. I am closing the issue. but I would suggest to update the readMe file for if someone wants to integrate AWSiOSSDKv2 in xcode 5.1 project so they may know how to do it.
